{
  "schema_version": "1.4.0",
  "id": "GHSA-9q62-r72g-pvv7",
  "modified": "2022-05-24T16:55:55Z",
  "published": "2022-05-24T16:55:55Z",
  "aliases": [
    "CVE-2019-16224"
  ],
  "details": "An issue was discovered in py-lmdb 0.97. For certain values of md_flags, mdb_node_add does not properly set up a memcpy destination, leading to an invalid write operation.",
  "severity": [

  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2019-16224"
    },
    {
      "type": "WEB",
      "url": "https://github.com/TeamSeri0us/pocs/tree/master/lmdb/lmdb%20initialization%20vuln"
    }
  ],
  "database_specific": {
    "cwe_ids": [

    ],
    "severity": null,
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2019-09-11T15:15:00Z"
  }
}